src/Etg.Yams/Etg.Yams.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Etg.Yams</id>
-    <version>1.0.14</version>
+    <version>1.1.0-test</version>
```

Comment:
No they should be replaced with the actual nuget version that will be published. I'll clean that up before merging, thanks.